docs/docs/_index.md

```diff
@@ -163,10 +190,15 @@ loadmodule /path/to/module/target/release/librejson.dylib
 
 In the above lines replace `/path/to/module/` with the actual path to the module's library.
 
-Alternatively, you can have Redis load the module using the following command line argument syntax:
+Alternatively, you can download and run RedisJSON from a precompiled binary:
 
-```bash
-~/$ redis-server --loadmodule ./target/release/librejson.so
+1. Download a precompiled version of RedisJSON from the [Redis download center](https://redis.com/download-center/modules/).
+
+1. Load RedisJSON:
+
+    ```sh
+    $ redis-server --loadmodule /path/to/library/librejson.so
+    ```
 ```
 
 Lastly, you can also use the [`MODULE LOAD`](/commands/module-load) command. Note, however, that `MODULE LOAD` is a **dangerous command** and may be blocked/deprecated in the future due to security considerations.
```

Comment:
There is a problem in the doc generator with backticks in links ([comment](https://github.com/RedisJSON/RedisJSON/pull/787/files#r942064010))
So either remove the backticks from within the square brackets of the link, or just use plain `MODULE LOAD` instead of a markdown link, and let the doc generator substitute it with the link to the command (there is some automation for backticks when generating docs)
```suggestion
Lastly, you can also use the [MODULE LOAD](/commands/module-load) command. Note, however, that `MODULE LOAD` is a **dangerous command** and may be blocked/deprecated in the future due to security considerations.
```

Reply:
The doc generator is run after pushing to master (see the [GH action](https://github.com/RediSearch/RediSearch/actions/workflows/trigger-build.yml))
So it could work for you now, but fail when generated and published.
I don't know whether it was fixed in the doc generator or not.